lms/djangoapps/learner_dashboard/views.py

```diff
@@ -66,6 +67,7 @@ def program_details(request, program_uuid):
 
     context = {
         'program_data': program_data,
+        'courses_progress': meter.progress(programs=meter.programs, count_only=False)[0],
```

Comment:
@MatthewPiatetsky not necessarily, I'm just asking you to find a way to avoid increasing the size of the response unnecessarily. That could mean popping the course data from the program data dict if it's not going to be messy.